src/panels/lovelace/common/data.ts

```diff
@@ -12,10 +13,17 @@ export const getCardConfig = (
 export const updateCardConfig = (
   hass: HomeAssistant,
   cardId: string,
-  config: any
+  config: LovelaceConfig | string,
+  configFormat: "json" | "yaml"
 ): Promise<void> =>
   hass!.callWS({
     type: "lovelace/config/card/update",
     card_id: cardId,
     card_config: config,
+    format: configFormat,
+  });
+
+export const migrateConfig = (hass: HomeAssistant): Promise<void> =>
+  hass!.callWS({
```

Comment:
Does TS actually complain that this might be undefined?

Reply:
Nope...